Code/Editor/ViewportManipulatorController.cpp

```diff
@@ -72,13 +74,23 @@ namespace SandboxEditor
                 const auto* position = event.m_inputChannel.GetCustomData<AzFramework::InputChannel::PositionData2D>();
                 AZ_Assert(position, "Expected PositionData2D but found nullptr");
 
+                if (m_captureStart)
+                {
+                    m_virtualNormalizedPosition = position->m_normalizedPosition;
+                    m_captureStart = false;
+                }
+                else
+                {
+                    m_virtualNormalizedPosition += position->m_normalizedPositionDelta;
+                }
+
                 AzFramework::WindowSize windowSize;
                 AzFramework::WindowRequestBus::EventResult(
                     windowSize, event.m_windowHandle, &AzFramework::WindowRequestBus::Events::GetClientAreaSize);
 
-                const auto screenPoint = AzFramework::ScreenPoint(
-                    aznumeric_cast<int>(position->m_normalizedPosition.GetX() * windowSize.m_width),
-                    aznumeric_cast<int>(position->m_normalizedPosition.GetY() * windowSize.m_height));
+                const auto screenPoint = AzFramework::ScreenPointFromVector2(AZ::Vector2(
+                    aznumeric_cast<int>(m_virtualNormalizedPosition.GetX() * windowSize.m_width),
+                    aznumeric_cast<int>(m_virtualNormalizedPosition.GetY() * windowSize.m_height)));
```

Comment:
update: Hmm I see this is a bit of an odd conversion, sorry for any confusion, I think this should ideally be:

```
const auto screenPoint = AzFramework::ScreenPointFromVector2(AZ::Vector2(
    (m_virtualNormalizedPosition.GetX() * aznumeric_cast<float>(windowSize.m_width)),
    (m_virtualNormalizedPosition.GetY() * aznumeric_cast<float>(windowSize.m_height))));
```